source/common/grpc/common.h

```diff
@@ -159,6 +160,11 @@ class Common {
    */
   static void prependGrpcFrameHeader(Buffer::Instance& buffer);
 
+  template <typename P> static bool parseBufferInstance(Buffer::InstancePtr&& buffer, P& proto) {
```

Comment:
This doesn't have to be templatized? You should be able to use `Protobuf::Message& proto`.

Reply:
Done.